roles/ansible-test-sanity/tasks/main.yml

```diff
@@ -0,0 +1,71 @@
+---
+
+###
+# Debug, list what we can find
+
+#- debug:
+#    var: hostvars
+
+
+# "{ ansible_user_dir }}/{{ zuul.project.src_dir }}" =  /home/zuul-worker/src/github.com/ansible-network/network-engine
+# "{{ ansible_path }}/test/runner/ansible-test"
+
+
+
+###
+# To test this Ansible Role using ansible-test we:
+# 1) Copy everything into "{{ ansible_path }}/role-to-test"
+# 2) Move modules, action plugins etc into the correct location under "{{ ansible_path }}/lib/ansible/" so they get tested correctly
+#    For this we just use shell cp. with_fileglob doesn't support running on remote-node
```

Comment:
You may want to update your comment to clarify step 1 uses `cp` and step 2 uses `mv.`